sdk/resourcemanager/azure-resourcemanager-keyvault/src/main/java/module-info.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.resourcemanager.keyvault {
+    requires transitive com.azure.resourcemanager.resources;
+    requires transitive com.azure.security.keyvault.keys;
+    requires transitive com.azure.security.keyvault.secrets;
+    requires com.azure.resourcemanager.authorization;
```

Comment:
/**
             * Specifies the Active Directory user this access policy is for.
             *
             * @param user the AD user object
             * @return the next stage of access policy definition
             */
            WithAttach<ParentT> forUser(ActiveDirectoryUser user);

Reply:
Updated by making `requires authorization` transitive.